src/main/java/com/animeviewing/application/entities/Animation.java

```diff
@@ -0,0 +1,63 @@
+package com.animeviewing.application.entities;
+
+import java.util.Optional;
+
+/**
+ * アニメーションエンティティ
+ */
+public class Animation {
+	/**
+	 * フィールド
+	 */
+	private AnimationId AnimationId;
```

Comment:
メソッド名は先頭文字列は小文字でお願いします。

```suggestion
	private AnimationId animationId;
```